examples/compiled/sha256sum.sh

```diff
@@ -14,70 +14,70 @@ defarr() { _malloc $1 $2; }
-  : $((_$((_k + 2)) = -1245643825))
-  : $((_$((_k + 3)) = -373957723))
-  : $((_$((_k + 4)) = 961987163))
-  : $((_$((_k + 5)) = 1508970993))
-  : $((_$((_k + 6)) = -1841331548))
-  : $((_$((_k + 7)) = -1424204075))
-  : $((_$((_k + 8)) = -670586216))
-  : $((_$((_k + 9)) = 310598401))
-  : $((_$((_k + 10)) = 607225278))
-  : $((_$((_k + 11)) = 1426881987))
-  : $((_$((_k + 12)) = 1925078388))
-  : $((_$((_k + 13)) = -2132889090))
-  : $((_$((_k + 14)) = -1680079193))
-  : $((_$((_k + 15)) = -1046744716))
-  : $((_$((_k + 16)) = -459576895))
-  : $((_$((_k + 17)) = -272742522))
-  : $((_$((_k + 18)) = 264347078))
-  : $((_$((_k + 19)) = 604807628))
-  : $((_$((_k + 20)) = 770255983))
-  : $((_$((_k + 21)) = 1249150122))
-  : $((_$((_k + 22)) = 1555081692))
-  : $((_$((_k + 23)) = 1996064986))
-  : $((_$((_k + 24)) = -1740746414))
-  : $((_$((_k + 25)) = -1473132947))
-  : $((_$((_k + 26)) = -1341970488))
-  : $((_$((_k + 27)) = -1084653625))
-  : $((_$((_k + 28)) = -958395405))
-  : $((_$((_k + 29)) = -710438585))
-  : $((_$((_k + 30)) = 113926993))
-  : $((_$((_k + 31)) = 338241895))
-  : $((_$((_k + 32)) = 666307205))
-  : $((_$((_k + 33)) = 773529912))
-  : $((_$((_k + 34)) = 1294757372))
-  : $((_$((_k + 35)) = 1396182291))
-  : $((_$((_k + 36)) = 1695183700))
-  : $((_$((_k + 37)) = 1986661051))
-  : $((_$((_k + 38)) = -2117940946))
-  : $((_$((_k + 39)) = -1838011259))
-  : $((_$((_k + 40)) = -1564481375))
-  : $((_$((_k + 41)) = -1474664885))
-  : $((_$((_k + 42)) = -1035236496))
-  : $((_$((_k + 43)) = -949202525))
-  : $((_$((_k + 44)) = -778901479))
-  : $((_$((_k + 45)) = -694614492))
-  : $((_$((_k + 46)) = -200395387))
-  : $((_$((_k + 47)) = 275423344))
-  : $((_$((_k + 48)) = 430227734))
-  : $((_$((_k + 49)) = 506948616))
-  : $((_$((_k + 50)) = 659060556))
-  : $((_$((_k + 51)) = 883997877))
-  : $((_$((_k + 52)) = 958139571))
-  : $((_$((_k + 53)) = 1322822218))
-  : $((_$((_k + 54)) = 1537002063))
-  : $((_$((_k + 55)) = 1747873779))
-  : $((_$((_k + 56)) = 1955562222))
-  : $((_$((_k + 57)) = 2024104815))
-  : $((_$((_k + 58)) = -2067236844))
-  : $((_$((_k + 59)) = -1933114872))
-  : $((_$((_k + 60)) = -1866530822))
-  : $((_$((_k + 61)) = -1538233109))
-  : $((_$((_k + 62)) = -1090935817))
-  : $((_$((_k + 63)) = -965641998))
+  : $((_$((_k + 0)) = 0x428a2f98))
+  : $((_$((_k + 1)) = 0x71374491))
+  : $((_$((_k + 2)) = 0xb5c0fbcf))
+  : $((_$((_k + 3)) = 0xe9b5dba5))
+  : $((_$((_k + 4)) = 0x3956c25b))
+  : $((_$((_k + 5)) = 0x59f111f1))
+  : $((_$((_k + 6)) = 0x923f82a4))
+  : $((_$((_k + 7)) = 0xab1c5ed5))
+  : $((_$((_k + 8)) = 0xd807aa98))
+  : $((_$((_k + 9)) = 0x12835b01))
+  : $((_$((_k + 10)) = 0x243185be))
+  : $((_$((_k + 11)) = 0x550c7dc3))
+  : $((_$((_k + 12)) = 0x72be5d74))
+  : $((_$((_k + 13)) = 0x80deb1fe))
+  : $((_$((_k + 14)) = 0x9bdc06a7))
+  : $((_$((_k + 15)) = 0xc19bf174))
+  : $((_$((_k + 16)) = 0xe49b69c1))
+  : $((_$((_k + 17)) = 0xefbe4786))
+  : $((_$((_k + 18)) = 0xfc19dc6))
```

Comment:
It's not perfect, as we don't keep track of the length of the literal, so leading zeros are not emitted.